keyboards/matchstickworks/southpad/rev2/info.json

```
@@ -0,0 +1,50 @@
+{
+    "keyboard_name": "SouthPad v2",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4D57",
+        "pid": "0x0002",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP13", "GP29", "GP28", "GP20", "GP19", "GP18"],
+        "cols": ["GP2", "GP4", "GP11", "GP12"]
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "label": "Bksp", "matrix": [0, 0], "x": 0.0, "y": 0.0 },
+                { "label": "=", "matrix": [0, 1], "x": 1.0, "y": 0.0 },
+                { "label": "(", "matrix": [0, 2], "x": 2.0, "y": 0.0 },
+                { "label": ")", "matrix": [0, 3], "x": 3.0, "y": 0.0 },
+                { "label": "-", "matrix": [1, 0], "x": 0.0, "y": 1.0 },
+                { "label": "*", "matrix": [1, 1], "x": 1.0, "y": 1.0 },
+                { "label": "/", "matrix": [1, 2], "x": 2.0, "y": 1.0 },
+                { "label": "Num Lock", "matrix": [1, 3], "x": 3.0, "y": 1.0 },
+                { "h": 2, "label": "+", "matrix": [2, 0], "x": 0.0, "y": 2.0 },
+                { "label": "9", "matrix": [2, 1], "x": 1.0, "y": 2.0 },
+                { "label": "8", "matrix": [2, 2], "x": 2.0, "y": 2.0 },
+                { "label": "7", "matrix": [2, 3], "x": 3.0, "y": 2.0 },
+                { "label": "6", "matrix": [3, 1], "x": 1.0, "y": 3.0 },
+                { "label": "5", "matrix": [3, 2], "x": 2.0, "y": 3.0 },
+                { "label": "4", "matrix": [3, 3], "x": 3.0, "y": 3.0 },
+                { "h": 2, "label": "Enter", "matrix": [4, 0], "x": 0.0, "y": 4.0 },
```

human readable formatting
```suggestion
                { "label": "+", "matrix": [2, 0], "x": 0.0, "y": 2.0, "h": 2 },
                { "label": "9", "matrix": [2, 1], "x": 1.0, "y": 2.0 },
                { "label": "8", "matrix": [2, 2], "x": 2.0, "y": 2.0 },
                { "label": "7", "matrix": [2, 3], "x": 3.0, "y": 2.0 },
                { "label": "6", "matrix": [3, 1], "x": 1.0, "y": 3.0 },
                { "label": "5", "matrix": [3, 2], "x": 2.0, "y": 3.0 },
                { "label": "4", "matrix": [3, 3], "x": 3.0, "y": 3.0 },
                { "label": "Enter", "matrix": [4, 0], "x": 0.0, "y": 4.0, "h": 2 },
```